fv3core/testing/validation.py

```diff
@@ -114,14 +122,22 @@ def _set_nans(self, tracers: Mapping[str, Quantity]):
     return SelectivelyValidatedTracerAdvection
 
 
-def get_compute_domain(
+def get_compute_domain_k_interfaces(
     instance,
 ) -> Tuple[Tuple[int, ...], Tuple[int, ...]]:
     origin = instance.grid.compute_origin()
     domain = instance.grid.domain_shape_compute(add=(0, 0, 1))
     return origin, domain
 
 
+def get_compute_domain_2d(
+    instance,
+) -> Tuple[Tuple[int, ...], Tuple[int, ...]]:
+    origin = (instance.grid.is_, instance.grid.js)
+    domain = (instance.grid.npx, instance.grid.npy)
```

Comment:
nit: if possible, use the compute/full origin/domain functions for this, since it describes the intent of these indices. I'm not actually sure if the domain argument here is accurate or not, because I can't remember if npx and npy are n_cells or n_cells + 1, which is kind of the issue.
```suggestion
    origin = instance.grid.compute_origin()[:2]
    domain = instance.grid.domain_shape_compute()[:2]
```

Reply:
npz and npy also are global tile indices, so this wouldn't have worked for 54 ranks